backend/inspirehep/logger/config.py

```diff
@@ -108,3 +129,7 @@ def log_error(
         task_args=args,
         task_kwargs=kwargs,
     )
+    task_name = sender.name if sender else "Unknown Task"
+    message = get_failure_message_by_task(task_name, exception, kwargs)
+    if message:
+        send_zulip_notification(message)
```

Comment:
my only concern is that it's still blocking. A solution will be to send the zulip notification as a task. wdyt?